python_src/src/lamp_py/performance_manager/gtfs_utils.py

```diff
@@ -209,46 +225,32 @@ def add_parent_station_column(
     return events_dataframe
 
 
-def remove_bus_records(
-    events_dataframe: pandas.DataFrame,
-    db_manager: DatabaseManager,
-) -> pandas.DataFrame:
+def rail_routes_from_filepath(
+    filepath: Union[List[str], str], db_manager: DatabaseManager
+) -> List[str]:
     """
-    remove all records from dataframe associated with bus trips
-
-    events_dataframe must have "static_version_key" and "route_id" columns
+    get a list of rail route_ids that were in effect on a given service date
+    described by a timestamp.
 
-    route_type == 3 from the routes table indicates bus service, drop all records
-    assocated with route_type == 3
+    the service_id_dates table is used to figure out the correct static version
+    key for a given service date, using the key with the max value (the keys
+    are also timestamps). then pull all the static routes with type
```

Comment:
update for new logic without `service_id_dates`

Reply:
done